src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -80,6 +84,8 @@ public void SaveCookieToken(HttpContext httpContext, string token)
             }
             SetCookiePath(httpContext, options);
 
+            _options.ConfigureCookieOptions?.Invoke(options);
```

Comment:
I call the new ConfigureCookieOptions last after everything else is set. If we don't want users to be able to override Secure or Path on the CookieOptions, we can move this up, though I don't see why we might want that.